test/tests/functional/pbs_provisioning.py

```diff
@@ -168,7 +168,7 @@ def test_app_provisioning(self):
         self.server.manager(MGR_CMD_SET, NODE, a, id=self.hostA)
 
         job = Job(TEST_USER1, attrs={ATTR_l: 'aoe=App1'})
-        job.set_sleep_time(1)
+        job.set_sleep_time(3)
```

Comment:
May be its better to remove this whole line, and let the sleep duration get set to the default 60 seconds.